crates/sui-framework/sources/governance/validator.move

```diff
@@ -531,43 +555,93 @@ module sui::validator {
         validate_metadata(&self.metadata);
     }
 
+    /// Update network address of this validator
+    public(friend) fun update_network_address(self: &mut Validator, net_address: vector<u8>) {
```

Comment:
This one seems to be forgotten? It should be renamed to `update_candidate_...` and we should assert the validator is preactive here too.

Reply:
Sigh...